test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/Metadata/ModelBinderAttributeTest.cs

```diff
@@ -49,5 +46,12 @@ public void BinderType_SettingBindingSource_OverridesDefaultCustomBindingSource(
             // Assert
             Assert.Equal(BindingSource.Query, source);
         }
+
+        private class FromQueryModelBinderAttribute : ModelBinderAttribute
+        {
+            // Not the perfect way to override this property since setting its value (possible when not using the class
+            // as an attribute) has no effect. base.BindingSource && BindingSource.Query may be right for some cases.
```

Comment:
Yeah, meant to say "Reviewers: ..." since users aren't going to look at this as an example of what they should or shouldn't do. Planned to remove.